sass/flag-icon-base.scss

```diff
@@ -0,0 +1,31 @@
+@mixin flag-icon-background {
+  background-size: contain;
+  background-position: 50%;
+  background-repeat: no-repeat;
+}
+
+.flag-icon {
+  @include flag-icon-background();
+  display: inline-block;
+  line-height: 1em;
+  position: relative;
+  width: (4 / 3) * 1em;
+
+  &:before {
+    content: '\00a0';
+  }
+
+  &.flag-icon-squared {
+    width: 1em;
+  }
+}
+
+@mixin flag-icon($country) {
+  .flag-icon-#{$country} {
+    background-image: url(#{$flag-icon-css-path}#{$flag-icon-rect-path}/#{$country}.svg);
+
+    &.flag-icon-squared {
+      background-image: url(#{$flag-icon-css-path}#{$flag-icon-square-path}/#{$country}.svg);
+    }
+  }
+}
```

Comment:
can you keep the same number of lines to have it one to one.. it will be easier for the conversations later.. and if @mixn flag-icon-background has to be first.. then change the order in the less file as well.. but either case I want them to have the same in terms of lines :)